src/main/kotlin/org/wycliffeassociates/otter/jvm/resourcecardpage/view/ProgressBanner.kt

```diff
@@ -0,0 +1,45 @@
+package org.wycliffeassociates.otter.jvm.resourcecardpage.view
+
+import com.jfoenix.controls.JFXCheckBox
+import javafx.scene.layout.Priority
+import javafx.scene.layout.VBox
+import javafx.scene.paint.Color
+import org.wycliffeassociates.otter.jvm.app.theme.AppStyles
+import org.wycliffeassociates.otter.jvm.resourcecardpage.styles.ProgressBannerStyles
+import tornadofx.*
+
+class ProgressBanner : VBox() {
+    init {
+        importStylesheet<ProgressBannerStyles>()
+
+        addClass(ProgressBannerStyles.progressBanner)
+        spacing = 10.0
+        hbox {
+            label("Translation Notes") {
+                graphic = AppStyles.tNGraphic()
+            }
+            region {
+                hgrow = Priority.ALWAYS
+            }
+            add(
+                    JFXCheckBox("Hide Completed").apply {
```

Comment:
excessive indent

Reply:
I wonder why ctrl+shift+L defaults to that much indentation...

Reply:
I wasn't sure because of IDEA's autoformatting, but Kotlin's [Coding Conventions](https://kotlinlang.org/docs/reference/coding-conventions.html#method-call-formatting) says just a single indent even for function arguments.

To fix this, File | Settings | Editor | Code Style | Kotlin | Set from... (top right) | Kotlin style guide